diesel_codegen/src/schema_inference/sqlite.rs

```diff
@@ -58,6 +58,40 @@ pub fn determine_column_type(cx: &mut ExtCtxt, attr: &ColumnInformation) -> P<as
     }
 }
 
+fn is_text(type_name: &str) -> bool {
+    type_name.contains("clob") ||
+        type_name.contains("text")
+}
+
+fn is_bool(type_name: &str) -> bool {
+    type_name == "boolean" ||
+        type_name.contains("tiny") &&
```

Comment:
I am bad at remembering boolean binding rules, do you think wrapping the ampersand expressions in parentheses would aid readability here?